src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/BaseProjectConfigurationDimensionProvider.cs

```diff
@@ -50,11 +50,8 @@ private async Task<ImmutableArray<string>> GetOrderedPropertyValuesAsync(Unconfi
 
             Assumes.NotNull(configuredProject);
 
-            return await ProjectAccessor.OpenProjectForReadAsync(configuredProject, evaluatedProject =>
-            {
-                // Need evaluated property to get inherited properties defines in props or targets.
-                return GetOrderedPropertyValues(evaluatedProject);
-            });
+            // Need evaluated property to get inherited properties defined in props or targets.
+            return await ProjectAccessor.OpenProjectForReadAsync(configuredProject, GetOrderedPropertyValues);
```

Comment:
This had to be manually fixed up.

Filed the bug as https://github.com/dotnet/roslyn/issues/60741